apps/previewgenerator.sh

```diff
@@ -48,14 +48,18 @@ else
     # reset the cronjob
     crontab -u www-data -l | grep -v 'preview:pre-generate'  | crontab -u www-data -
     # Remove apps
-    APPS=(ffmpeg php-imagick libmagickcore-6.q16-3-extra)
+    APPS=(php-imagick libmagickcore-6.q16-3-extra)
     for app in "${APPS[@]}"
     do
         if is_this_installed "$app"
         then
             apt purge "$app" -y
         fi
     done
+    if is_this_installed ffmpeg && ! is_app_installed integration_whiteboard
+    then
+        apt purge ffmpeg -y
```

Comment:
Hmm, I why does that package need to be removed? And why in previewgenerator?

Reply:
Because ffmpeg also gets installed by the previewgenerator script (becausee it is needed for Movie previews)

Reply:
Wouldn't this cause a conflict if both Preview Generator and Whiteboard are installed at the same time?

Reply:
No, I don't see the problem here:
If you uninstall previewgenerator and integration_whiteboard is installed, it doesn't remove ffmpeg.
If you uninstall whiteboard, and movie previews are enabled, it doesn't remove ffmpeg.

Or are you refering to something else?

Reply:
As I see it:

1. Before the change `ffmpeg` was always removed. Probably beacuse it doesn't work with previewgenerator(?)
2. Now, you only purge it if the whiteboard app isn't installed

Result of 2:
If the Whiiteborad app s installed, ffpmeg will not be removed thus previewgenerator will not work?

Reply:
Since ffmpeg is like a library, there is no problem if two apps are using it at the same time...

Reply:
Now I get it :)
No, ffmpeg gets removed when reinstalling previewgenerator but it needed for movie previews here as well. So in conclusion: we don't remove it here because it is incompatible but because we reinstall the previewgenerator!

Reply:
OK, thanks!

Please solve the conflicts and I'll merge.

Reply:
(I was lazy and didn't read everything) :man_technologist: